src/components/ProfilingToolMenu/BaseProfilingToolMenu.tsx

```diff
@@ -49,16 +49,16 @@ const newFileName = `Profile_trace_for_${pkg.version}.cpuprofile`;
 
 function BaseProfilingToolMenu({isProfilingInProgress = false, pathToBeUsed, displayPath}: BaseProfilingToolMenuProps) {
     const styles = useThemeStyles();
-    const [pathIOS, setPathIOS] = useState('');
+    const [filePath, setFilePath] = useState('');
     const [sharePath, setSharePath] = useState('');
     const [totalMemory, setTotalMemory] = useState(0);
     const [usedMemory, setUsedMemory] = useState(0);
     const {translate} = useLocalize();
 
     // eslint-disable-next-line @lwc/lwc/no-async-await
     const stop = useCallback(async () => {
-        const path = await stopProfiling(getPlatform() === CONST.PLATFORM.IOS);
-        setPathIOS(path);
+        const path = await stopProfiling(getPlatform() === CONST.PLATFORM.IOS || getPlatform() === CONST.PLATFORM.WEB, newFileName);
```

Comment:
Do we need the condition getPlatform() === CONST.PLATFORM.WEB here? 

For stopProfiling true/false both values are downloading the file in the Downloads folder for me.

Reply:
@jayeshmangwani without that condition it'll always throw a warning:

https://github.com/margelo/react-native-release-profiler/blob/582a72f627bb1d1c8f1ef94c0e7031a4568c2de3/src/index.web.tsx#L86-L90

Reply:
Okay, gotcha. I was suggesting that to avoid the platform-specific code in the file, though we can pass the saveToDownloads value from the platform-specific files

Reply:
> though we can pass the saveToDownloads value from the platform-specific files

Should I re-work this part as you are suggesting? 👀

Reply:
I think it's fine here. We were already using getPlatform() === CONST.PLATFORM.IOS. Sorry for the confusion. Please leave it as it is for now.